libuuid/uuid.c

```diff
@@ -87,31 +101,35 @@ void uuid_unparse(const uuid_t uu, char *out)
 
 int uuid_parse(const char *in, uuid_t uu)
 {
-	int i, ret;
+	int ret;
 	unsigned char val;
-	const char *temp;
```

Comment:
Modification of function parameters is by some considered a bad practice

Reply:
I will opt for leaving the temp variable and not changing the `in` paremeter

Reply:
@agkaminski But you use it e.g. here [string.c](https://github.com/phoenix-rtos/libphoenix/blob/master/string/string.c).